build/index.d.ts

```diff
@@ -0,0 +1,32 @@
+export interface InitParams {
```

Comment:
build artifacts (`build/*`) should be gitignored

Reply:
Apologies for the late reply, that change has been undone